lib/config.js

```diff
@@ -1,8 +1,21 @@
 /*!
- * Copyright (c) 2012-2024 Digital Bazaar, Inc.
+ * Copyright 2012 - 2024 Digital Bazaar, Inc.
```

Comment:
Why add spaces here and elsewhere?
```suggestion
 * Copyright 2012-2024 Digital Bazaar, Inc.
```

Reply:
Because the `reuse` command uses spaces... I find it annoying also, but didn't think it worth fighting the bot about.

If you feel it is...we can fork https://github.com/fsfe/reuse-tool until they make that an option.

Reply:
ISO 8601 apparently uses a `/` between dates to specify a range--and thankfully also allows for `--`.

So if we (and/or the reuse project) wanted to make this super parseable, I'd vote for the `--` use.

Per https://en.wikipedia.org/wiki/ISO_8601#Time_intervals

Reply:
We can't do what we want because the robot is broken?
https://github.com/fsfe/reuse-tool/blob/main/src/reuse/_util.py#L388
Grepping through thousands of installed files, I only find ~4 files (or transformations thereof) with that spaced out pattern.  I'd say the robot is wrong in this case.  And to be fair, mostly everyone does single years.  But under 1% is a strange choice for them to make.

Reply:
> So if we (and/or the reuse project) wanted to make this super parseable, I'd vote for the `--` use.
> 

That would be even *more* of an outlier from common use than ` - `.

Reply:
Agreed. How would you like to fix this? I don't feel it's worth continuing to drag out the relicensing or avoid the automation. So, if you want to fight the whitespace fight, I'd take it upstream--or propose a cleanup script to make it match your preference: https://github.com/fsfe/reuse-tool/issues

Reply:
@davidlehn would you prefer I fork the tool and fix this whitespace? or can you live with the spaces?